core-bundle/src/Mailer/ContaoMailer.php

```diff
@@ -115,4 +104,36 @@ private function setFrom(Email $message): void
             $message->sender($from);
         }
     }
+
+    /**
+     * Copy of AbstractFragmentController::getPageModel.
```

Comment:
I've seen this in extensions as well. Maybe it should be reusable? /cc @contao/developers

Reply:
Call is on Thursday - if you mark it up for discussion, I think it would be in time for the next bugfix release?

Reply:
I don't really think this makes sense though. I have the exact same code in my app as well. For an event listener.

Reply:
As discussed in the Contao call, adding a new service in Contao 4.13.36 does not make sense, especially since we do not need the service in Contao 5 anymore (because `pageModel` is always an object there).